tests/ui/inherent_to_string.rs

```diff
@@ -0,0 +1,84 @@
+#![warn(clippy::inherent_to_string)]
+//#![deny(clippy::inherent_to_string_shadow)]
```

Comment:
Please uncomment this
```suggestion
#![deny(clippy::inherent_to_string_shadow)]
```
or remove it
```suggestion
```